config/routes.rb

```diff
@@ -0,0 +1,70 @@
+Rails.application.routes.draw do
+  root 'parks#index'
+
+  resources :ranger
+
+  delete '/ranger/:id' => 'ranger#destroy'
+
+  get '/ranger/:id' => 'ranger#show'
+  post '/ranger/:id' => 'ranger#show'
+
+  post '/ranger/new' => 'ranger#create'
```

Comment:
The add ranger route isn't working because you are posting to the route '/ranger/new' and you have a post route above it to catch the id 'ranger/:id'. this is catching the post to new and literally taking the word 'new' out of teh adress and trying to use it as an id.